src/components/Backdrop/index.js

```diff
@@ -19,17 +21,21 @@ export default class Backdrop extends Component {
   static defaultProps = {
     overlay: true,
     overlayProps: {},
-    onRequestClose: null
+    onRequestClose: null,
+    isVisible: true,
+    animated: false
```

Comment:
I think we should default this to `true`

Reply:
Actually we can remove it.

Reply:
It's fine to leave